### ISSUE TITLE
Desktop animations not working while an orbtk application is open

### ISSUE THREAD

**Issue by Ayush1325** [redox-os/orbtk#454]
Whenever I launch an orbtk application or any of the examples, my normal desktop applications seem to stop working. And everything is fine as soon as I close the application

# Steps to reproduce the behavior:
1. Run an orbtk example.
2. Switch virtual desktop or open the menu.

# Expected behavior
See my beautiful desktop animation.

# System Information
 - OS: Fedora Workstation 34 KDE Spim
 - Desktop: KDE Plasma 5.22.4
 - Graphics Platform: X11
 - GPU: GTX 1050Ti
 - Nvidia Drivers Version: 470.57.02

# Additional context
It seems that orbtk application somehow conflicts with the KDE compositor. 
There is no unusual CPU and GPU utilization, in fact, the application itself runs quite well, so I am not sure what the problem is.
Can anyone else confirm or deny this issue?

**Comment by adi-g15** [comment]
Yes, it stops transparency and animations (both tasks of the compositor ig) on my arch+KDE system too.

#### System Information
* OS - Arch Linux (linux 5.13.13-arch1-1)
* DE - KDE Plasma (5.22.5-1)
* X11

This is the screen recording of the same:

https://user-images.githubusercontent.com/37269665/132528241-a6df6e2a-a6b3-46b2-adf2-e71232f3a98e.mp4

**Comment by adi-g15** [comment]
An Update on this: **It works fine with Wayland**

Plasma + Wayland

**Comment by Ayush1325** [comment]
@adi-g15 Cool. Will just have to wait till Wayland starts working well with Nvidia.

**Comment by XakepSDK** [comment]
It disables compositor as if it was a game. This is handled by SDL2. I think OrbTk should tell SDL that app is not a game.

**Comment by trickypr** [comment]
> It disables compositor as if it was a game. This is handled by SDL2. I think OrbTk should tell SDL that app is not a game.

It seems that `SDL_HINT_VIDEO_X11_NET_WM_BYPASS_COMPOSITOR` would have to be set to `"0"` using [`sdl2::hint::set`](https://docs.rs/sdl2/0.35.1/sdl2/hint/fn.set.html). I suspect that this would have to be changed in orbclient on the Redox gitlab, so I am not able to create a merge request. 

Also see [this issue](https://github.com/mosra/magnum/issues/184#issuecomment-425952900).

**Comment by trickypr** [comment]
As mentioned in #471, you can disable compositor blocking for development purposes, however this is not ideal for a production application:

![Screenshot_20211227_184016](https://user-images.githubusercontent.com/23250792/147447889-be83aa40-c901-42bd-8cea-2f423626bb38.png)
